lib/connection/connection_config.js

```diff
@@ -52,7 +53,8 @@ const DEFAULT_PARAMS =
   'includeRetryReason',
   'disableQueryContextCache',
   'retryTimeout',
-  'forceGCPUseDownscopedCredential'
+  'forceGCPUseDownscopedCredential',
+  'fetchOnlyNull',
```

Comment:
I changed the option name to 'treatNullAsNull'. Is it okay?

Reply:
Better. What do you think about 'representNullsAsStringNulls' with default value true. Isn't it better?

Reply:
> I edited. As the parameter name is 'representNullAsStringNull', so the default value will be true because this is the original behavior, and the driver only returns null value as null when this is false.